Guardfile

```diff
@@ -0,0 +1,55 @@
+# Defines the matching rules for Guard.
+guard :minitest, spring: true, all_on_start: false do
+  watch(%r{^test/(.*)/?(.*)_test\.rb$})
+  watch('test/test_helper.rb') { 'test' }
+  watch('config/routes.rb')    { integration_tests }
+  watch(%r{^app/models/(.*?)\.rb$}) do |matches|
+    "test/models/#{matches[1]}_test.rb"
+  end
+  watch(%r{^app/controllers/(.*?)_controller\.rb$}) do |matches|
+    resource_tests(matches[1])
+  end
+  watch(%r{^app/views/([^/]*?)/.*\.html\.erb$}) do |matches|
+    ["test/controllers/#{matches[1]}_controller_test.rb"] +
+    integration_tests(matches[1])
```

Comment:
__[rubocop]__
* Use 2 (not 0) spaces for indenting an expression spanning multiple lines.